xwiki-commons-tools/xwiki-commons-tool-verification-resources/src/main/resources/checkstyle.xml

```diff
@@ -101,7 +101,7 @@
         List, ArrayList, Deque, Queue, LinkedList, Set, HashSet, SortedSet, TreeSet, Map, HashMap, SortedMap, TreeMap, Collections, Collection, Arrays,
         ConcurrentHashMap, ConcurrentLinkedDeque, ConcurrentLinkedQueue, DelayQueue, FutureTask, LinkedBlockinDeque, LinkedBlockinQueue, Phaser, PriorityBlockingQueue, RejectedExecutionException, SynchronousQueue,
         Logger, LoggerFactory, StringUtils, IOUtils,
-        Inject, Named, Component, Role, Singleton, Unstable"/>
+        Inject, Named, Component, Role, Singleton, Unstable, Override, Deprecated, SafeVarargs, SuppressWarnings"/>
```

Comment:
You sure those count in ClassFanOutComplexity ? Sounds pretty weird for Checkstyle to count Deprecated.

Reply:
Now no, but after this [PR](https://github.com/checkstyle/checkstyle/pull/5122) will be count.

Reply:
OK. I would not have required any configuration for this kind of annotation but I guess you discussed this already :)